mlflow/keras.py

```diff
@@ -146,21 +146,21 @@ def log_model(keras_model, artifact_path, conda_env=None, **kwargs):
               keras_model=keras_model, conda_env=conda_env, **kwargs)
 
 
-def _load_model(model_file):
+def _load_model(model_file, *args, **kwargs):
```

Comment:
Could we add a unit test to `test_keras_model_export` that confirms that these extra kwargs are in fact passed through the keras properly on load?

Reply:
I don't know how to do this without mocking keras, since these arguments only make a difference when custom metrics are used.